adele-payment-manager/src/main/java/com/microservicesteam/adele/payment/paypal/PaypalProxy.java

```diff
@@ -0,0 +1,24 @@
+package com.microservicesteam.adele.payment.paypal;
+
+import org.springframework.stereotype.Component;
+
+import com.paypal.api.payments.Payment;
+import com.paypal.base.rest.APIContext;
+import com.paypal.base.rest.PayPalRESTException;
+
+@Component
+public class PaypalProxy {
+
+    private final APIContext apiContext;
+
+    public PaypalProxy(PaypalConfig.PaypalProperties paypalProperties) {
+        apiContext = new APIContext(
```

Comment:
I think this is still not good enough, as Paypal documentation states that a new `APIContext` is needed for every payment call, while here `PaypalProxy.apiContext` is bound to the lifecycle of the proxy.
`PaypalProxy.create(...)` should create it's own api context object at every call.